tests/Feature/MovieIndexTest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Tests\Feature;
+
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Tests\TestCase;
+
+class MovieIndexTest extends TestCase
+{
+    /**
+     * Tests for Movies.Index
+     *
+     * @return void
+     */
+    public function test_movie_index_screen_can_render()
+    {
+        $response = $this->get('/movies');
+
+        $response->assertStatus(200);
+    }
```

Comment:
You can just chain this one ;D and use assertOk() instead of the assertStatus(200). Just for consistency